README.md

```diff
@@ -28,12 +28,12 @@ Links between docs can be achieved by using a hash symbol (#), plus the markdown
 
 ### Alert Boxes
 
-Alert boxes are achieved by using a block quote that has one of 'warn', 'danger' or 'info' on the first line.
+Alert boxes are achieved by using a block quote that has one of 'warn', 'danger', or 'info' on the first line.
```

Comment:
It’s also fine to not use Oxford commas, so I don’t think this change is necessary.